app/javascript/src/Policies/actions/PolicyChain.jsx

```diff
@@ -40,13 +39,13 @@ export function loadChainSuccess (payload: Array<ChainPolicy>): LoadChainSuccess
   return { type: 'LOAD_CHAIN_SUCCESS', payload }
 }
 
-export type LoadChainErrorAction = { type: 'LOAD_CHAIN_ERROR', payload: Object }
-export function loadChainError (payload: Object): LoadChainErrorAction {
+export type LoadChainErrorAction = { type: 'LOAD_CHAIN_ERROR', payload: {} }
+export function loadChainError (payload: {}): LoadChainErrorAction {
```

Comment:
The payload is literally `{}`: https://github.com/3scale/porta/blob/68cadd5ea7061bb03278f13eb38edc279b72e918/app/javascript/src/Policies/middleware/PolicyChain.jsx#L45

This is one of the things that should be improved but for now it's inaccurate to say the payload is `Object`.

Reply:
Here it makes sense, since as you commented, the payload is literally `{}`. Add a `$FlowFixMe` and for the rest just leave `Object`, because the warning acts as a TODO it self.

Reply:
Yes, I think we shouldn't hide the warnings

Reply:
@didierofrivia 
I don't think a `$FixFlowMe` is needed this particular case, if that's what you mean, since there's already a TODO.
As for the rest, oki doki.